USAGE.md

```diff
@@ -1,370 +1,707 @@
+| [Matrix](#matrices)              | `x-mitre-matrix` |
+| [Tactic](#tactics)               | `x-mitre-tactic` |
+| [Technique](#techniques)         | `attack-pattern` |
+| [Sub-technique](#sub-techniques) | `attack-pattern` where `x_mitre_is_subtechnique = true` |
+| [Procedure](#procedures)         | `relationship` where `relationship_type = "uses"` and `target_ref` is an `attack-pattern` |
+| [Mitigation](#mitigations)       | `course-of-action` |
+| [Group](#groups)                 | `intrusion-set`  |
+| [Software](#software)            | `tool` or `malware` |
 
+Two additional object types are found in the ATT&CK catalog:
+ 
+| STIX object type | About |
+|:-----------------|:------|
+| `identity`       | Referenced in the `created_by_ref` of all objects to state that the MITRE Corporation created the object |
+| `marking-definition` | Referenced in the `object_marking_refs` of all objects to express the MITRE Corporation copyright |
+
+## Common departures from the STIX spec
+
+There are three general ways that ATT&CK departs from the stock STIX 2.0 format:
+- Custom object types. Object types prefixed with `x-mitre-`, e.g `x-mitre-matrix`, are custom STIX types not defined in the stock STIX spec. They follow the general [STIX Domain Object pattern](https://docs.oasis-open.org/cti/stix/v2.0/csprd01/part2-stix-objects/stix-v2.0-csprd01-part2-stix-objects.html#_Toc476230920) but describe concepts not covered by generic STIX2.0 types.
+
+- Extensions of existing object types. Fields not found in the stock STIX 2.0 spec are prefixed with `x_mitre_`, e.g `x_mitre_platforms` in `attack-patterns`. 
+    
+    All objects except relationships can have the following extended properties applied:
+
+    | field | type | description |
+    |:------|:-----|:------------|
+    | `x_mitre_revoked` | boolean | If true, the object has been replaced by another object. A relationship will exist describing the replacing object where the `source_ref` is the ID of this object and the `target_ref` is the ID of the replacing object. See [working with deprecated and revoked objects](#Working-with-deprecated-and-revoked-objects) for more information on revoked objects.
+    | `x_mitre_version` | string | The version of the object in format `major.minor` where `major` and `minor` are integers. ATT&CK increments this version number when the object content is updated. |
+    | `x_mitre_contributors` | string[] | People and organizations who have contributed to the object. | 
+
+- New relationship types. Unlike custom object types and extended fields, custom relationship types are **not** prefixed with `x_mitre_`. You can find a full list of relationship types in the [relationships](#relationships) section, which also mentions whether the type is a default STIX type.
+
+Please see also the STIX documentation on [customizing STIX](https://docs.oasis-open.org/cti/stix/v2.0/csprd01/part1-stix-core/stix-v2.0-csprd01-part1-stix-core.html#_Toc476227365).
```

Comment:
By the end of this section I now understand what's going on, but I feel like that understanding came out of order? Like you should first say what's different from the STIX spec (i.e. custom objects) and then clarify in the ATT&CK concept to STIX object mapping what's generic and what's custom?

Reply:
I'm hoping that the other rewrites I'm doing in the 2nd draft will help the reader better understand and distinguish stock STIX vs our extensions.